src/Symfony/Component/Dotenv/Tests/DotenvTest.php

```diff
@@ -148,6 +148,13 @@ public function getEnvData()
 
     public function testLoad()
     {
+        unset($_ENV['FOO']);
+        unset($_ENV['BAR']);
+        unset($_SERVER['FOO']);
+        unset($_SERVER['BAR']);
+        putenv('FOO');
+        putenv('BAR');
```

Comment:
I think we should also clean the environment first in case someone has defined the `$FOO` or `$BAR` env variable which would make the assertions fail (and that's how I discovered the missing cleanup).